### PR TITLE
Login Page is updated Successfully

### PR DESCRIPTION
**Pull request by HeartTick**
#257 @dohinafs Login page has been updated successfully.
image

### CODE REVIEW

**Review by Altair-05 — APPROVED** [review]

**Comment by HeartTick** [comment]
Hello @Altair-05 can you please go through the code and merge the code so that i can work on further Issues in this webpage .

File: index.html [hunk 1]
```diff
@@ -427,12 +431,17 @@ <h2>Login to Your Account</h2>
...
             <input type="email" placeholder="Email" required>
+<!-- <<<<<<< Icecream_Website -->
             <h3>Password :-</h3>
              <br>
             <input type="password" placeholder="Password" required class="inputFields">
             <button type="submit">Login</button>
             
           <p>Don't have an account? <a href="signup.html">Sign up</a></p>
+=======
+            <input type="password" class="pass-btn" placeholder="Password" required>
```
Comment (Altair-05) [inline index.html:442]: why the password placeholder is displayed again in this section?

**Review by Altair-05 — CHANGES_REQUESTED** [review]
@HeartTick 
image

**Comment by HeartTick** [comment]
Can you check once again . because i can't see that line in my code editor.

**Comment by HeartTick** [comment]
image
This is the ouput i am getting when i am running the code and i checked the code as well there is no sush line exist at that position.

**Comment by Altair-05** [comment]
@HeartTick, please check the deploy preview. You’ll see the issue there

**Comment by HeartTick** [comment]
Ss I can see the error in preview but I can able to see the error in my local machine.
Can u guide me to resolve this issue.

**Review by dohinafs — COMMENTED** [review]
This is how it is appearing in the deployment preview:
image
Make necessary changes to fix above and if possible add more padding below the cone.

**Comment by HeartTick** [comment]
Can I close this pull request and make a new one . Is that ok for you

**Comment by dohinafs** [comment]
Yes you can

**Pull request closed without merging**